Assets/HoloToolkit/Utilities/Scripts/Solvers/SolverMomentumizer.cs

```diff
@@ -0,0 +1,101 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+using UnityEngine;
+using System.Collections;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    ///   Momentumizer solver applies accel/velocity/friction to simulate momentum for an object being moved by other solvers/components
+    /// </summary>
+    public class SolverMomentumizer : Solver
+	{
+		[Tooltip("Friction to slow down the current velocity")]
+		public float resistance = 0.99f;
+		[Tooltip("Apply more resistance when going faster- applied resistance is resistance * (velocity ^ reisistanceVelPower)")]
+		public float resistanceVelPower = 1.5f;
+		[Tooltip("Accelerate to goal position at this rate")]
+		public float accelRate = 10f;
+		[Tooltip("Apply more acceleration if farther from target- applied accel is accelRate + springiness * distance")]
+		public float springiness = 0;
+
+		[Tooltip("Instantly maintain a constant depth from the view point instead of simulating Z-velocity")]
+		public bool SnapZ = true;
+
+		private Vector3 velocity;
+
+		public override void SolverUpdate()
+		{
+            CalculateMomentum();
```

Comment:
formatting in this class seems off

Reply:
I'm not sure why this is happening, it looks fine in VS. I reformatted/tabified - hopefully that should fix it.